### ISSUE TITLE
This package can working with .godotenv ?

### ISSUE THREAD

**Issue by thuandoan274** [kelseyhightower/envconfig#129]
I wanna set my env in file .env using package https://github.com/joho/godotenv
Can I use this package to get env from .env

**Comment by hypnoglow** [comment]
@thuandoan274 Yes, these packages can easily be used together. See an [example](https://github.com/hypnoglow/chronologist/blob/master/cmd/chronologist/config.go).
